### PR TITLE
Use sane defaults on config.php

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
Fixes #72

File: config/config.php [hunk 1]
```diff
@@ -1,69 +1,66 @@
...
-            ]
-        ];
-        return $config;
-    }
+            'files' => [
+                'avatar_extentions' => [ 'jpg', 'png', 'jpeg' ],
+                'flag_extention' => 'png'
+            ],
+            'paths' => [
+                'avatar_path' => 'sample_path',
```
Comment (dionyziz) [inline config/config.php]: Use a better default

File: config/config.php [hunk 2]
```diff
@@ -1,69 +1,66 @@
...
+                'user' => 'travis',
+                'pass' => '',
+                'dbname' => 'endofcodes_test'
+            ],
+            'files' => [
+                'avatar_extentions' => [ 'jpg', 'png', 'jpeg' ],
+                'flag_extention' => 'png'
+            ],
+            'paths' => [
+                'avatar_path' => 'sample_path',
```
Comment (dionyziz) [inline config/config.php]: Use a better defaut; "avatars"?

File: config/config.php [hunk 3]
```diff
@@ -1,69 +1,66 @@
...
-                'email' => 'team@endofcodes.com',
-                'base' => 'http://localhost/endofcodes/',
-                'root' => '/var/www/endofcodes/'
+    return [
+        'development' => [
+            'db' => [
+                'host' => 'sample_host',
+                'user' => 'sample_user',
+                'pass' => 'sample_pass',
+                'dbname' => 'sample_dbname'
```
Comment (dionyziz) [inline config/config.php]: Use sane defaults here. Some sane defaults are 'localhost' for host, 'endofcodes' for user and dbname. Password of course can't have a sane default, so this is fine.

**Pull request merged**